test/C/src/federated/FeedbackDelay5.lf

```diff
@@ -0,0 +1,53 @@
+target C {
+  timeout: 900 ms
```

Comment:
Why not just 1 second like the other tests?

Reply:
This is so that PhysicalPlant can be imported unmodified. It expects 10 inputs, and if we set to timeout to 1s, it will get 11 and flag an error.